src/test/ui/hygiene/extern-prelude-from-opaque-fail.stderr

```diff
@@ -5,7 +5,7 @@ LL |     use my_core;
    |         ^^^^^^^
    |         |
    |         no `my_core` in the root
-   |         help: a similar name exists in the module: `my_core`
+   |         help: a similar name exists in the module (notice the capitalization): `my_core`
```

Comment:
The capitalisation is irrelevant in this case.

Reply:
These errors look bad in more ways than one, but fair.